robot_skills/src/robot_skills/world_model_ed.py

```diff
@@ -195,7 +193,11 @@ def get_entity(self, id, parse=True):
         return entities[0]
 
     def get_entity_info(self, id):
-        return self._ed_entity_info_query_srv(id=id, measurement_image_border=20)
+        try:
```

Comment:
Might be out of scope but in general I actually don't like catching all exceptions at this level: it implies you always have to check all return values (is it empty? is it 'None'?). And if so, what do I return if to the caller above? IMHO, exceptions should be caught at the level that they can be handled appropriately.

Reply:
True, but I found that all service calls in this file were wrapped in a try/except. Therefore I also wrapped this service call.